README.md

```diff
@@ -240,3 +243,21 @@ describe('ComponentToTest', () => {
   }
 }
 ```
+
+
+## Release cycle:
```

Comment:
This is more about "how to release" and less the "release cycle", which would be more about when and how often releases are published.

Reply:
neu: 
## Release a new version: